buildSrc/src/main/java/saros/gradle/eclipse/CategoryPublisher.java

```diff
@@ -0,0 +1,29 @@
+package saros.gradle.eclipse;
+
+import com.diffplug.gradle.FileMisc;
+import com.diffplug.gradle.eclipserunner.EclipseApp;
+import com.diffplug.gradle.pde.EclipseRelease;
+import com.diffplug.gradle.pde.PdeInstallation;
+import java.io.File;
+
+public class CategoryPublisher extends EclipseApp {
+
+  private final String eclipseVersion;
+
+  public CategoryPublisher(String eclipseVersion) {
+    super("org.eclipse.equinox.p2.publisher.CategoryPublisher");
```

Comment:
Is there an external constant we can use for this? That would avoid having to hard-code such a specific external dependency in a string. If not, I would still suggest moving it to a local constant.

Reply:
No, this is [defined by eclipse](https://wiki.eclipse.org/Equinox/p2/Publisher#Category_Publisher), not the Goomph plugin.